onecodex/cli.py

```diff
@@ -93,7 +108,7 @@ def analyses(ctx, analyses):
 @onecodex.command('classifications')
 @click.option("--read-level", 'readlevel', is_flag=True,
               help=OPTION_HELP['readlevel'])
-@click.option("--read-level-path", 'readlevel_path',
+@click.option("--read-level-path", 'readlevel_path', type=click.Path(),
```

Comment:
Is this needed for the underlying `scripts`? Fine either way.

Reply:
It's not required, I just noticed and thought it might make sense to add the type.

Reply:
Nope that's fine!